.github/workflows/tests.yml

```diff
@@ -0,0 +1,58 @@
+name: test documentation
+
+on: [ push, pull_request ]
+
+jobs:
+  test-documentation:
```

Comment:
I would like to keep the two in separate files as we do in the other manuals

Reply:
@linawolf Agreed, please have a look again.